### DISCUSSION TITLE
How to expand the height of q-tab-panel/q-scroll-area to full height of a col-md-10 parent?

### DISCUSSION THREAD

**Discussion by prototyperspective** [quasarframework/quasar#15194] [General - Components / Directives / etc] [answered]
Why do I have to set the height of q-scroll-area when height of parent is set and col and col-md-2 is used as below? I already tried using class="full-height". How can I make the q-tab-panel expand to full height? The code is like this:

```
<div class="row col-12">
    <q-card class="column col-12" style="height: 250px">
      <q-tabs class="col-md-2">...</q-tabs>
      <q-tab-panels class="col-md-10">
         <q-tab-panel>...</q-tab-panel>
         <q-tab-panel>
           <q-tab-panels class="col-md-10">
              <q-tab-panel class="full-height">
                 <q-scroll-area class="full-height">
                     This content area should expand to the full height of the col-md-10 parent
                 <q-scroll-area>
               </q-tab-panel>
               <q-tab-panel>...</q-tab-panel>
           </q-tab-panels>
          <q-tabs class="col-md-2">...</q-tabs>
         </q-tab-panel>
      </q-tab-panels>
```
It only works when I specify a height of the q-tab-panel (200px) just above the scrollarea but it should be as flexible as possible (and be using the col classes; I hope the code is fine otherwise so there's no better way to have nested qtabpanels with multiple tab-rows).

There's a codepen below. Maybe the problem has with col-md-2 qtabs not taking up space before and after the qtabpanel.

**Comment by pdanpdan** [comment]
Make a codepen

**Reply by prototyperspective** [reply]
It doesn't display the blue and green spaces yet but here's the link already https://codepen.io/prototyperspective/pen/qByZmjE and I don't know why it doesn't load the Quasar things and display the tabs properly, I thought all js resources were loaded

**Comment by pdanpdan** [comment, marked as answer]
https://codepen.io/pdanpdan/pen/XWBdZqL?editors=1010

Why would you expect your codepen to work if you did not import any js and css files? :))))

**Reply by prototyperspective** [reply]
Thanks so much! This works perfectly.  
Sorry for the codepen issues, don't know why I only imported the js files but not the css.  
This could also be useful as an example for how to make q-cards q-tab-panels with two rows (top and bottom) of tabs.
Have a great new year!
